src/main/resources/db/changelog/changes/v1.1.0/harvested-views.xml

```diff
@@ -48,4 +48,15 @@
     </createView>
   </changeSet>
 
+  <changeSet id="create_view_custom_fields" author="kjain@ebsco.com">
+    <preConditions onFail="CONTINUE">
+      <tableExists tableName = "custom_fields" schemaName="${tenant_id}_mod_users"/>
+    </preConditions>
+    <createView
+      replaceIfExists="true"
+      viewName="user_custom_fields">
```

Comment:
Nit: we should put some kind of prefix on this view name, to indicate that it's not one of the core FQM DB objects. `src_` could make sense, since this view is definitely source data. But it's not like other source data, since it defines columns instead of actual data, so maybe something else...